app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -1106,7 +1067,6 @@ export default class SampleViewV2 extends React.Component {
               reportPresent={!!reportMetadata.reportReady}
               sample={sample}
               view={view}
-              minContigReads={selectedOptions.minContigReads}
```

Comment:
and this

Reply:
yes. Did you clean up the header on the previous PR?